app/templates/withnav_template.html

```diff
@@ -16,6 +16,9 @@
         {% endif %}
       {% endif %}
       <div class="navigation-service-name govuk-!-font-weight-bold">
+        {% if not current_service.active %}
+        [SUSPENDED]
+        {% endif %}
         {{ current_service.name }}
         {% if current_service.has_permission('broadcast') %}
```

Comment:
You could use the same styling as the training mode label here. It has a similar meaning – the service is prevented from doing any live stuff, be that broadcasts or emails etc.

It also removes the slightly incongruous combination of _[SUSPENDED] Service name LIVE_ for broadcast services.
```suggestion
        {{ current_service.name }}
        {% if not current_service.active %}
        <span class="navigation-service-type navigation-service-type--training">Suspended</span>
        {% elif current_service.has_permission('broadcast') %}
```

Reply:
I was thinking that a service being suspended is a separate dimension from whether it's live (and which broadcast provider was chosen). That said, I agree the practical impact overlaps with the concept of "training" and "live" in terms of what the service can do.

I've made the change you suggested, with a tweak to apply it to _all_ services (not just broadcast ones), since it makes sense in all cases. I've added a couple of screenshots to show the before/after too.

What do you think?